esiclient/v1/node_network.py

```diff
@@ -97,7 +97,13 @@ def get_parser(self, prog_name):
             '--port',
             dest='port',
             metavar='<port>',
-            help=_("Attach to this port (name or UUID).")
+            help=_("Attach to this neutron port (name or UUID).")
```

Comment:
Whats the difference between neutron port and port?

Reply:
Neutron and Ironic both have their own ports, which aren't quiiiite the same thing. I added 'neutron' here to clarify that.